Sources/Queue/AsyncQueue.swift

```diff
@@ -1,10 +1,15 @@
 import Foundation
+import Combine
```

Comment:
What is this import doing?

Reply:
Sorry, that's a leftover! I was going to use the Cancellable protocol which is defined in `Combine` but I changed my mind 😃 I'll remove it

Reply:
Fixed

Reply:
I had a feeling that was it!